stripe-core/src/main/java/com/stripe/android/core/StripeError.kt

```diff
@@ -80,5 +80,11 @@ constructor(
      *
      * [doc_url](https://stripe.com/docs/api/errors#errors-doc_url)
      */
-    val docUrl: String? = null
+    val docUrl: String? = null,
+
+    /**
+     * Internal list of extra fields related to the error.
+     * Note - value type is ignored and always parsed as string (true -> "true")
+     */
+    internal val extraFields: Map<String, String>? = null
```

Comment:
Assuming this should remain internal as it's not [documented](https://stripe.com/docs/api/errors).